### PR TITLE
DOC : Update the pandas.Period.week docstring

### PR DESCRIPTION
**Pull request by IHackPy**
```
# Period.week
################################################################################
######################## Docstring (pandas.Period.week) ########################
################################################################################

Get the total weeks on the Period falls in.

Returns
-------
int 

See Also
--------
Period.dayofweek : Get the day component of the Period.
Period.weekday : Get the day component of the Period.

Examples
--------
>>> p = pd.Period("2018-03-11")
>>> p.week
10

################################################################################
################################## Validation ##################################
################################################################################
```

### CODE REVIEW

File: pandas/_libs/tslibs/period.pyx [hunk 1]
```diff
@@ -1305,6 +1305,24 @@ cdef class _Period(object):
 
     @property
     def week(self):
+        """
+        Get the total weeks on the Period falls in.
```
Comment (TomAugspurger) [inline pandas/_libs/tslibs/period.pyx]: Total weeks -> "week of year"?

We'll want to keep an eye on https://github.com/pandas-dev/pandas/issues/20324 for this.

Comment (HimanshuAwasthi95) [reply pandas/_libs/tslibs/period.pyx]: "Get the week of the year on the given Period"?

sure will check #20324

Comment (TomAugspurger) [reply pandas/_libs/tslibs/period.pyx]: Can you update the summar?

**Review by jorisvandenbossche — APPROVED** [review]

**Comment by jorisvandenbossche** [comment]
@HimanshuAwasthi95 Thanks!

**Comment by HimanshuAwasthi95** [comment]
thanks @TomAugspurger & @jorisvandenbossche

**Pull request merged**